sound/soc/sof/Makefile

```diff
@@ -11,7 +11,7 @@ snd-sof-y +=	ipc3.o ipc3-loader.o ipc3-topology.o ipc3-control.o ipc3-pcm.o\
 endif
 ifneq ($(CONFIG_SND_SOC_SOF_IPC4),)
 snd-sof-y += ipc4.o ipc4-loader.o ipc4-topology.o ipc4-control.o ipc4-pcm.o\
-		ipc4-mtrace.o ipc4-telemetry.o
+		ipc4-mtrace.o ipc4-telemetry.o ipc4-debug-slot-debugfs.o
```

Comment:
this implements read-only debugfs files, right? Should we add "ro" to the file name to make this clear?

Reply:
If we ever need read-write access, wouldn't it be logical to extend this implementation without a need to rename the implementation file? How about changing function name sof_ipc4_create_debug_slot_debugfs_node() to sof_ipc4_create_debug_slot_ro_debugfs_node() ?